### ISSUE TITLE
Add example or demo

### ISSUE THREAD

**Issue by gberger** [nonplus/angular-ui-router-uib-modal#4]
Can you add a live example/demo (i.e. in Plunkr)? It would really help when figuring out how to use it and what the code should look like.

**Comment by gberger** [comment]
Btw, great job on this little lib! It's very useful

**Comment by anx-ckreuzberger** [comment]
Agreed, a live example would be really cool to have! Also great job on this "little helper", my router code is now so much more dry!

**Comment by s0549096** [comment]
Would also love an example! :)
